cmd/do-agent/config.go

```diff
@@ -164,6 +164,20 @@ func initDecorator() decorate.Chain {
 	return chain
 }
 
+// initAggregatorSpecs initializes the field aggregation specifications.
+// The map's key is the prometheus metric name to aggregate over, and the value is the label to aggregate away.
```

Comment:
Do we maybe want this to be a `map[string][]string` to allow for dropping multiple labels?

Reply:
I was saying this AM that we could, but figured we could easily do it later when we had a use case for it and instead opted to keep it as simple as possible for now

Reply:
:+1: